.gitignore

```diff
@@ -3,6 +3,10 @@
 # C extensions
 *.so
 
+# vim temporal files
```

Comment:
please don't do this and introduce unnecessary change, if your editor generates extra temporary files then ignore them with a global git ignore like https://help.github.com/articles/ignoring-files/